src/library/librarytablemodel.cpp

```diff
@@ -101,3 +105,13 @@ TrackModel::CapabilitiesFlags LibraryTableModel::getCapabilities() const {
             | TRACKMODELCAPS_CLEAR_BEATS
             | TRACKMODELCAPS_RESETPLAYED;
 }
+
+void LibraryTableModel::setSort(int column, Qt::SortOrder order)
+{
```

Comment:
nit (sorry): We place { always at the end of a line. 
https://www.mixxx.org/wiki/doku.php/coding_guidelines